.github/ISSUE_TEMPLATE/support.yaml

```
@@ -0,0 +1,108 @@
+name: 🚢 Support
+description: Support needed with Prometheus Operator
+labels: ["kind/support", "needs-triage"]
+body:
+- type: markdown
+  attributes:
+    value: |
+      **Note:** This repository is about prometheus-operator itself, if you have questions about:
+            - kube-prometheus setup, go to [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus)
+            - helm installation, go to [kube-prometheus-stack-helm-chart](https://github.com/prometheus-community/helm-charts)
+- type: markdown
+  attributes:
+    value: |
+      Before filing a request, please be sure you have checked existing documentation in [prometheus-operator.dev](https://prometheus-operator.dev)?
```

```suggestion
      Are you aware that we have documentation at [our website](https://prometheus-operator.dev)? There's a chance you'll find answers faster over there 🙂
```

I think we could have a less aggressive tone here and just mention that reading the docs might be faster

---

That sounds better 🙂